encord/client_metadata_schema.py

```diff
@@ -5,6 +5,16 @@
 
 
 def set_client_metadata_schema_from_dict(api_client: ApiClient, json_dict: Dict[str, orm.ClientMetadataSchemaTypes]):
```

Comment:
Moreso a point for @sergei-encord. The below code handles schema types presented as strings. So technically the interface to this function should be changed and reduces a dependency of user provided ormTypes. Thoughts?

Reply:
We can just import ClientMetadataSchemaTypes to this file, so users would be able to access it as 
``` 
from encord.client_metadata_schema import ClientMetadataSchemaTypes
```
without looking into the ORM. 
Does it sound reasonable? I'll do separate pr to fix that though